data/ikea/Wall plug

```diff
@@ -0,0 +1,18 @@
+{
+ 	"measure_description": "Manually measured. Transcribed from https://github.com/bramstroker/homeassistant-powercalc/discussions/207#discussioncomment-1484985",
+ 	"measure_method": "manual",
+ 	"measure_device": "Shelly Plug S",
+ 	"name": "Ikea Wall plug",
```

Comment:
```suggestion
 	"name": "TRÅDFRI Wireless control outlet, smart (NL)",
```
Or maybe 'EU plug'?

Reply:
In Home assistant it's called "TRADFRI control outlet" so maybe go with that?

Reply:
I'm ok with both, but I think I prefer the suggestion of @CloCkWeRX as this name is only used in the supported models listing. Makes sense to be as specific as possible there so the user can make sure his device is actually the same as in the powercalc db.